src/components/AddPaymentCard/PaymentCardForm.tsx

```diff
@@ -0,0 +1,332 @@
+import {useRoute} from '@react-navigation/native';
+import React, {useCallback, useRef, useState} from 'react';
+import type {ReactNode} from 'react';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import AddressSearch from '@components/AddressSearch';
+import CheckboxWithLabel from '@components/CheckboxWithLabel';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import Hoverable from '@components/Hoverable';
+import * as Expensicons from '@components/Icon/Expensicons';
+import type {AnimatedTextInputRef} from '@components/RNTextInput';
+import StateSelector from '@components/StateSelector';
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/AddDebitCardForm';
+import PaymentCardCurrencyModal from './PaymentCardCurrencyModal';
+
+type PaymentCardFormProps = {
+    shouldShowPaymentCardForm?: boolean;
+    showAcceptTerms?: boolean;
+    showAddressField?: boolean;
+    showCurrencyField?: boolean;
+    showStateSelector?: boolean;
+    isDebitCard?: boolean;
+    addPaymentCard: (values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_DEBIT_CARD_FORM>, currency?: ValueOf<typeof CONST.CURRENCY>) => void;
+    submitButtonText: string;
+    /** Custom content to display in the footer after card form */
+    footerContent?: ReactNode;
+    /** Custom content to display in the footer before card form */
+    headerContent?: ReactNode;
+};
+
+function IAcceptTheLabel() {
+    const {translate} = useLocalize();
+
+    return (
+        <Text>
+            {`${translate('common.iAcceptThe')}`}
+            <TextLink href={CONST.TERMS_URL}>{`${translate('common.addCardTermsOfService')}`}</TextLink> {`${translate('common.and')}`}
+            <TextLink href={CONST.PRIVACY_URL}> {` ${translate('common.privacyPolicy')} `}</TextLink>
+        </Text>
+    );
+}
+
+function IAcceptDebitTheLabel() {
+    const {translate} = useLocalize();
+
+    return (
+        <Text>
+            {`${translate('common.iAcceptThe')}`}
+            <TextLink href={CONST.TERMS_URL}>{`${translate('common.expensifyTermsOfService')}`}</TextLink>
+        </Text>
+    );
+}
```

Comment:
I am pretty sure we want to update the "I accept the" portion of the form to be identical for debit card and payment card (so completely remove `IAcceptDebitTheLabel()` variant and both forms would have the user accept both the terms of service and privacy policy).

cc @trjExpensify though, can you confirm I'm remembering correctly?

Reply:
Yeah, there isn't a good reason to have one include both ToS and privacy policy, and the other just ToS - so let's align the two and drop this debit card variant so it's consistent everywhere. 👍